syft/grid/network.py

```diff
@@ -164,3 +158,12 @@ def __repr__(self):
             list(self._worker.models.keys()),
             list(self._connection_handler.nodes),
         )
+
+    def peers(self, key):
```

Comment:
Just an ideia! Take peers as a @property here and overload the WebRTCManager \__getitem__ function. With this it would be possible to do things like me.peers['peer_name'] too.

Reply:
Done!